.gitignore

```diff
@@ -1,46 +1,42 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
-# compiled output
+# Compiled output
 /dist
 /tmp
 /out-tsc
-# Only exists if Bazel was run
 /bazel-out
 
-# dependencies
+# Node
```

Comment:
The `node_modules` folder is actually all the dependencies of the project and hence `dependencies` make more sense than `Node`. Please revert this change.

Reply:
Change is reverted.